docs/operations.md

```diff
@@ -115,7 +115,7 @@ while the 'Severity' criterium is as follows:
 * **Major** - The system has a problem that degrades the service and must be addressed
 * **Warning** - It is happening something that must be notified
 
-## <a name="errorcode"/> Error naming code
+## Error naming code
```

Comment:
Now the index link to this subsection does not work

Reply:
Fixed - facd80d - ToC regenerated